mkfs.f2fs: add page (#12564)
tldr-pages/tldr: pages/linux/mkfs.f2fs.md

# mkfs.f2fs

> > Creates a F2FS filesystem inside a partition.

Reviewer: ```suggestion
> Creates an F2FS filesystem inside a partition.
```

> More information: <https://manned.org/mkfs.f2fs>.

> - Create an F2FS filesystem inside partition 1 on device b (`sdb1`):

Reviewer 2: ```suggestion
- Create an F2FS filesystem on a block device (`/dev/sdb1`):
```

I think better to keep it generic and call it *block device*. NVMe drives have very different naming like `nvme0`. The partition can also be made be a logical volume like `/dev/vg0/lv0`.

Author: I copied the wording from mkfs.ext4: https://github.com/tldr-pages/tldr/blob/main/pages/linux/mkfs.ext4.md
Now that I look at them. Almost all of the mkfs. commands have the same wording.

Reviewer 2: In that case, I guess okay to leave it as it is. Maybe later we can update all the `mkfs.*` pages and refer to partitions as block devices.

`sudo mkfs.f2fs {{/dev/sdb1}}`

> - Create an F2FS filesystem with a volume-label:

Reviewer: ```suggestion
- Create an F2FS filesystem with a volume label:
```
